pkg/cmd/package/nuget/create/create.go

```diff
@@ -0,0 +1,314 @@
+				return err
+			}
+		}
+
+		if opts.ReleaseNotes.Value == "" {
+			if err := opts.Ask(&surveyext.OctoEditor{
+				Editor: &survey.Editor{
+					Message: "Nuspec release notes",
+					Help:    "The release notes to include in the Nuspec file.",
+				},
+				Optional: true,
+			}, &opts.ReleaseNotes.Value); err != nil {
+				return err
+			}
+		}
+
+		if opts.ReleaseNotesFile.Value == "" {
+			if err := opts.Ask(&survey.Input{
+				Message: "Nuspec release notes file",
+				Help:    "A path to a release notes file whose contents will be included in the Nuspec file's release notes.",
+			}, &opts.ReleaseNotesFile.Value); err != nil {
+				return err
+			}
+		}
+	}
+
+	return nil
+}
+
+func applyDefaultsToUnspecifiedPackageOptions(opts *NuPkgCreateOptions) error {
+	if opts.Version.Value == "" {
+		opts.Version.Value = pack.BuildTimestampSemVer(time.Now())
+	}
+
+	if opts.BasePath.Value == "" {
+		opts.BasePath.Value = "."
+	}
+
+	if opts.OutFolder.Value == "" {
+		opts.OutFolder.Value = "."
+	}
+
+	if len(opts.Include.Value) == 0 {
+		opts.Include.Value = append(opts.Include.Value, "**")
+	}
+
+	if len(opts.Author.Value) > 0 {
+		if opts.Description.Value == "" {
+			opts.Description.Value = "A deployment package created from files on disk."
+		}
+	}
+
+	return nil
+}
+
+func getReleaseNotesFromFile(filePath string) (string, error) {
+	_, err := os.Stat(filePath)
+	if err != nil {
+		return "", err
+	}
+
+	notes, err := os.ReadFile(filePath)
+	if err != nil {
+		return "", err
+	}
+
+	return string(notes), nil
+}
+
+func shouldGenerateNuSpec(opts *NuPkgCreateOptions) bool {
+	return opts.Description.Value != "" ||
+		opts.Title.Value != "" ||
+		opts.ReleaseNotes.Value != "" ||
+		opts.ReleaseNotesFile.Value != "" ||
+		!util.Empty(opts.Author.Value)
+}
+
+func GenerateNuSpec(opts *NuPkgCreateOptions) (string, error) {
+
+	if opts.Description.Value == "" {
+		return "", errors.New("description is required when generating nuspec metadata")
+	}
+	if len(opts.Author.Value) == 0 {
+		return "", errors.New("at least one author is required when generating nuspec metadata")
+	}
+
+	releaseNotes := opts.ReleaseNotes.Value
+	if opts.ReleaseNotesFile.Value != "" {
+		if releaseNotes != "" {
+			return "", errors.New(`cannot specify both "Nuspec release notes" and "Nuspec release notes file"`)
+		}
+
+		notes, err := getReleaseNotesFromFile(opts.ReleaseNotesFile.Value)
+		releaseNotes = notes
+		if err != nil {
+			return "", err
+		}
+	}
+
+	filePath := filepath.Join(opts.BasePath.Value, opts.Id.Value+".nuspec")
+
+	var sb strings.Builder
+	sb.WriteString(`<?xml version="1.0" encoding="utf-8"?>` + "\n")
+	sb.WriteString(`<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">` + "\n")
+	sb.WriteString("  <metadata>\n")
+	sb.WriteString("    <id>" + opts.Id.Value + "</id>\n")
+	sb.WriteString("    <version>" + opts.Version.Value + "</version>\n")
+	sb.WriteString("    <description>" + opts.Description.Value + "</description>\n")
```

Comment:
Need to escape the string values here, otherwise you'll get a broken XML file if someone puts a `<` in their version/description/author/etc